llvm/lib/Target/RISCV/GISel/RISCVCallLowering.cpp

```diff
@@ -164,44 +194,101 @@ struct RISCVIncomingValueHandler : public CallLowering::IncomingValueHandler {
 
   void assignValueToReg(Register ValVReg, Register PhysReg,
                         CCValAssign VA) override {
-    // Copy argument received in physical register to desired VReg.
-    MIRBuilder.getMBB().addLiveIn(PhysReg);
-    MIRBuilder.buildCopy(ValVReg, PhysReg);
+    markPhysRegUsed(PhysReg);
```

Comment:
Correct me if I'm wrong, but it looks like this plus a couple lines below in terms of changing  markPhysRegUsed definitions is an NFC-esq refactoring?  Can it be landed separately?

Reply:
It is in the patch as a separate commit. I'll commit it separately and not squash it.